build_projects/dotnet-host-build/build.ps1

```diff
@@ -8,7 +8,7 @@ param(
     [string]$Architecture="x64",
     [string]$TargetArch="",
     [string]$ToolsetDir="",
-    [string]$Framework="netcoreapp1.0",
+    [string]$Framework="netcoreapp1.1",
```

Comment:
Right, but that is publishing the build project itself, it doesn't control anything else in the build process that I can see. That line is analogous to line 122 in this file, mentioned above. I think we should still be publishing the build exe itself for `netcoreapp1.0` (well, it really doesn't matter what we use, as long as it works).